Entities/Common/GUI/DefaultActorHUD.as

```diff
@@ -89,7 +89,12 @@ void onRender(CSprite@ this)
 	renderBackBar(ul, dim.x, 1.0f);
 	u8 bar_width_in_slots = blob.get_u8("gui_HUD_slots_width");
 	f32 width = bar_width_in_slots * 40.0f;
-	renderFrontStone(ul + Vec2f(dim.x + 40, 0), width, 1.0f);
+
+	// additional space for drawing resupply icon
```

Comment:
The HUD should only be enlarged when the resupply mechanism actually exists. That could also be factored out to a function named `shouldRenderResupplyIndicator`.

Reply:
oversight, added